env0/credentials.go

```diff
@@ -0,0 +1,47 @@
+package env0
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/google/uuid"
+)
+
+func getCredentialsByName(name string, prefix string, meta interface{}) (client.Credentials, error) {
```

Comment:
Common helper functions for credentials (can be used by GCP and Azure in the future). (In case import is added to them as well).